src/test/scala/org/ergoplatform/network/TransactionSpecification.scala

```diff
@@ -0,0 +1,80 @@
+package org.ergoplatform.network
+
+import java.nio.ByteBuffer
+
+import org.ergoplatform.utils.ErgoPropertyTest
+import scorex.util.encode.Base16
+
+class TransactionSpecification extends ErgoPropertyTest with DecodingUtils {
+
+  property("transaction specification parser") {
+    /** Header flag to indicate that constant segregation should be applied. */
+    val ConstantSegregationFlag: Byte = 0x10
+
+    /** Header flag to indicate that whole size of ErgoTree should be saved before tree content. */
+    val SizeFlag: Byte = 0x08
```

Comment:
These flags are defined in ErgoTree from Sigma. It make sense to refer to either sigma definition or to ErgoTree Specification document.